src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -2,6 +2,10 @@
 
 from __future__ import print_function
 
+import sys
+
```

Comment:
Add a comment linking to #9239 and a TODO indicating it can be removed when #7091 is fixed.

Reply:
Done (in internal merge of this PR).